examples/plotters_combined.py

```diff
@@ -36,7 +36,7 @@
 from pimoroni_ltr559 import LTR559
 from pimoroni_pms5003 import PMS5003
 
-#print("(", gc.mem_free(), ")")
+print(f"initial memory: {gc.mem_free():,}")
```

Comment:
Formatting options not necessary.
```suggestion
print(f"initial memory: {gc.mem_free()}")
```

Reply:
those are thousand separators for the printed display

Reply:
Ah, right.